templates/Gemfile.erb

```diff
@@ -35,10 +35,11 @@ group :test do
   gem 'capybara-webkit', '>= 1.0.0'
   gem 'database_cleaner'
   gem 'launchy'
+  gem 'rubocop', require: false
+  gem 'scss_lint', require: false
```

Comment:
I think these 2 should be moved to the `dev + test` group. Rubocop was already here, but it was probably wrong